README.Rmd

```diff
@@ -0,0 +1,191 @@
+---
+output: github_document
+author: "Mauro Lepore, `r Sys.time()`."
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%",
+  cache = TRUE
+)
+
+devtools::load_all()
+```
+
+# PACTA_analysis
+
+The goal of this repository is to assess how well a porfolio aligns with climate
+goals. 
+
+This documents targets internal 2DII users and developers. It provides
+reproducible examples of what you can achieve with the code in this repository
+and friends. You may use it as a guide to run your own analyses, or as an
+integration test to ensure code changes preserve the behavior documented here.
+Readers outside 2DII may instead see other related work:
+[transitionmonitor.com](https://platform.transitionmonitor.com/start),
+[r2dii.data](https://github.com/2DegreesInvesting/r2dii.data),
+[r2dii.match](https://github.com/2DegreesInvesting/r2dii.match), and
+[r2dii.analysis](https://github.com/2DegreesInvesting/r2dii.analysis).
+
+You may want to analyze a single portfolio and investor, or multiple ones. This document details two workflows:
+
+* __Single-inputs workflow__: When the number of portfolios and investors is
+exactly one, we use a specific set of scripts optimized for this purpose. This
+workflow is typically used to generate interactive reports. Because this
+workflow is available [online](https://platform.transitionmonitor.com/start), it
+has been referred to as "the online workflow", and the scripts as "the web tool
+scripts". This is misleading because this workflow can run offline too.
+* __Multiple-outputs workflow__: When the number of portfolios and/or investors
+is not exactly one, we use a different set of optimized scripts. This workflow
+is typically used to produce static (.pdf) reports. By contrast with the
+so-called "online" workflow, this one has been usually referred to as "the
+offline workflow"; again, this is misleading.
+
+Both workflows include three steps:
+
+1. Clean portfolio: Clean the input portfolio and merge in financial data, to
+categorize each holding and identify whether it's equity (EQ) or corporate bonds
+(CB).
+2. Run Analysis: Merge portfolios with asset-level data and scenarios, then
+group results at company, portfolio and regional level.
+3. Present results: Present the results in a clear output format.
```

Comment:
Personally, I would like to detach this 3rd step from the process in the future. What I imagine is that running the PACTA analysis generates results (meaning data that has been merged and certain calculations done) based on the input portfolio and various options. After that, a user (or web server) may want to use those results to generate an interactive report, or maybe an Executive Summary PDF, or maybe just one interactive alignment plot, or maybe just do some manual analysis with it in R, or maybe some combination of those. But that's the point where users branch out into different things.

Reply:
Agree. Right now I'm focusing on documenting the current state of things so I lean to leave it as is for now.

https://github.com/2DegreesInvesting/PACTA_analysis/issues/417